lmfit/minimizer.py

```diff
@@ -647,16 +645,17 @@ def penalty(self, fvars):
             sum-of-squares, but can be replaced by other options.
 
         """
-        if self.result.method in ['brute', 'shgo', 'dual_annealing']:
-            apply_bounds_transformation = False
-        else:
-            apply_bounds_transformation = True
+        apply_bounds_transformation = self.result.method not in [
+            'brute',
+            'shgo',
+            'dual_annealing',
+        ]
```

Comment:
please revert this, this reverse logic makes it much less readable at first glance...